### ISSUE TITLE
Keystroke for read selected text is not working in laptop layout

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8504]
### Steps to reproduce:
1. Open NVDA with keyboard layout set on "laptop"
2. open a text editor and write down two words
3. Select one word and press NVDA+shift+s

### Actual behavior:
NVDA does not read anything.

### Expected behavior:
NVDA should read the selected text.

#### NVDA Installed/portable/running from source:
Both

#### NVDA version:
2018.2.1

#### Windows version:
Windows 10 (1803 update)

#### Name and version of other software in use when reproducing the issue:
MS Office, Internet browsers etc.

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA?
no

**Comment by didiercolle** [comment]
what text editor are you using?
I cannot reproduce this issue in notepad, notepad++, wordpad

**Comment by Adriani90** [comment]
in any text editor, NVDA+shift+s does not announce selected text. I can reproduce it even when I disable all my add-ons. I have also reinstalled NVDA and still having this issue.

**Comment by Adriani90** [comment]
Ok I tested on Windows 8.1 with NVDA 2017.3 and it works properly. It must be something wrong with my personal Laptop. I would close this as worksforme but I don't have writing rights. So closing this without Label.
